sdk/js/src/index.ts

```diff
@@ -222,24 +228,60 @@ export function send_rpc_request<T = any>(endpoint: string, path: string, payloa
   })
 }
 
-export class TappdClient {
+export interface TlsKeyOptions {
+  path?: string;
+  subject?: string;
+  altNames?: string[];
+  usageRaTls?: boolean;
+  usageServerAuth?: boolean;
+  usageClientAuth?: boolean;
+}
+
+export class DstackClient {
   private endpoint: string
+  private logger: Console
 
-  constructor(endpoint: string = '/var/run/tappd.sock') {
+  constructor(endpoint: string = '/var/run/dstack.sock') {
+    this.logger = console
     if (process.env.DSTACK_SIMULATOR_ENDPOINT) {
-      console.debug(`Using simulator endpoint: ${process.env.DSTACK_SIMULATOR_ENDPOINT}`)
+      this.logger.debug(`Using simulator endpoint: ${process.env.DSTACK_SIMULATOR_ENDPOINT}`)
```

Comment:
Better switch to `INFO` because when working on customer support, we need the line to check if they built the image incorrectly.

Reply:
Done